usim/_primitives/context.py

```diff
@@ -115,36 +138,77 @@ def do(
                 await containing_scope
                 print('... scope has finished')
 
-            with Scope() as scope:
+            async with Scope() as scope:
                 scope.do(graceful(scope))
 
-        All `volatile` activities are aborted at the end of the scope,
-        after all non-`volatile` activities have finished.
-        Aborting ``volatile` activities is not graceful:
+        All ``volatile`` activities are aborted at the end of the scope,
+        after all non-``volatile`` activities have finished.
+        Aborting ``volatile`` activities is not graceful:
         :py:class:`GeneratorExit` is raised in the activity,
-        which must exit without `await`\ ing or `yield`\ ing anything.
+        which must exit without ``await``\ ing or ``yield``\ ing anything.
         """
-        child_task = Task(payload)
+        assert after is None or at is None,\
+            "schedule date must be either absolute or relative"
+        assert after is None or after > 0,\
```

Comment:
I would not understand the error message when entering `0` for `after`. So we should consider including `0`.

```suggestion
        assert after is None or after >= 0,\
```

Reply:
I've added ticket #35 for this.